### PR TITLE
Add 'Are you finding it hard to afford rent, your mortgage or bills?' question

### PR DESCRIPTION
**Pull request by Rosa-Fox**
affording-rent-mortgage-bills

### CODE REVIEW

**Comment by Rosa-Fox** [comment]
going to change `affording` to `afford` to align with `afford-food`

File: config/routes.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@
     get "/still-working", to: "still_working#show"
     post "/still-working", to: "still_working#submit"
 
-    # Question: Are you worried about going into work because you live with someone vulnerable to coronavirus?
+    # Question: Are you worried about going into work because you live with someone vulnerable
```
Comment (brucebolt) [inline config/routes.rb:33]: I'm not sure this was intentional?

File: config/locales/en.yml [hunk 2]
```diff
@@ -157,6 +157,12 @@ en:
           - "Yes"
           - "No"
           - "Not sure"
+      afford_rent_mortgage_bills:
```
Comment (brucebolt) [inline config/locales/en.yml:160]: I'd suggest moving this up to line 120, so the questions remain in the same order as the Figma prototype (https://www.figma.com/file/tibZIxmeYW0Z3qEL4CEF72/Triage-tool---prototypes?node-id=953%3A2).

File: config/routes.rb (hunk 1 above)
Comment (Rosa-Fox) [reply config/routes.rb]: ah no, error when rebasing! Thanks, have amended.

File: config/locales/en.yml (hunk 2 above)
Comment (Rosa-Fox) [reply config/locales/en.yml]: Thanks! Have amended.

**Pull request merged**